src/main_menu.cpp

```diff
@@ -962,7 +962,8 @@ bool main_menu::new_character_tab()
         // First load the mods, this is done by
         // loading the world.
         // Pick a world, suppressing prompts if it's "play now" mode.
-        WORLD *world = world_generator->pick_world( true, sel2 == 3 || sel2 == 4 );
+        const bool is_play_now = sel2 == 3 || sel2 == 4;
+        WORLD *world = world_generator->pick_world( !is_play_now, is_play_now );
```

Comment:
` WORLD *worldfactory::pick_world( bool show_prompt, bool empty_only )`

Reply:
Would it be easy to write a test for this so it doesn't regress again?

Reply:
Depends what you mean by easy :)

My gut says that I would have to fix this same bug a few more times before it would be worth it to write a good test for this.

Reply:
Makes sense.